aeron-driver/src/main/c/agent/aeron_driver_agent.c

```diff
@@ -1288,67 +1653,84 @@ void aeron_driver_agent_log_dissector(int32_t msg_type_id, const void *message,
+                aeron_driver_agent_dissect_log_header(hdr->time_ns, msg_type_id, length, length),
+                hdr->stream_id,
+                hdr->id,
+                hdr->channel_length,
+                channel);
             break;
         }
 
-        case AERON_RAW_LOG_CLOSE_OP:
+        case AERON_DRIVER_EVENT_REMOVE_IMAGE_CLEANUP:
         {
-            aeron_driver_agent_raw_log_op_header_t *hdr = (aeron_driver_agent_raw_log_op_header_t *)message;
-
+            aeron_driver_agent_remove_resource_cleanup_t *hdr = (aeron_driver_agent_remove_resource_cleanup_t *)message;
+            const char *channel = (const char *)message + sizeof(aeron_driver_agent_remove_resource_cleanup_t);
             fprintf(
                 logfp,
-                "[%s] RAW_LOG_CLOSE %p = %d\n",
-                aeron_driver_agent_dissect_timestamp(hdr->time_ms),
-                (void *)hdr->raw_log.raw_log_close.addr,
-                hdr->raw_log.raw_log_close.result);
+                "%s: sessionId=%d, streamId=%d, id=%" PRId64 ", uri=%*s\n",
+                aeron_driver_agent_dissect_log_header(hdr->time_ns, msg_type_id, length, length),
+                hdr->session_id,
+                hdr->stream_id,
+                hdr->id,
+                hdr->channel_length,
+                channel);
             break;
         }
 
-        case AERON_RAW_LOG_FREE_OP:
+        case AERON_DRIVER_EVENT_SEND_CHANNEL_CREATION:
+        case AERON_DRIVER_EVENT_SEND_CHANNEL_CLOSE:
+        case AERON_DRIVER_EVENT_RECEIVE_CHANNEL_CREATION:
+        case AERON_DRIVER_EVENT_RECEIVE_CHANNEL_CLOSE:
         {
-            aeron_driver_agent_raw_log_op_header_t *hdr = (aeron_driver_agent_raw_log_op_header_t *)message;
-
+            aeron_driver_agent_on_endpoint_change_t *hdr = (aeron_driver_agent_on_endpoint_change_t *)message;
             fprintf(
                 logfp,
-                "[%s] RAW_LOG_FREE %p = %s\n",
-                aeron_driver_agent_dissect_timestamp(hdr->time_ms),
-                (void *)hdr->raw_log.raw_log_free.addr,
-                hdr->raw_log.raw_log_free.result ? "true" : "false");
+                "%s: localData: %s, remoteData: %s, ttl: %d\n",
+                aeron_driver_agent_dissect_log_header(hdr->time_ns, msg_type_id, length, length),
+                dissect_sockaddr((const struct sockaddr *) &hdr->local_data),
+                dissect_sockaddr((const struct sockaddr *) &hdr->remote_data),
+                hdr->multicast_ttl);
             break;
         }
 
-        case AERON_UNTETHERED_SUBSCRIPTION_STATE_CHANGE:
+        case AERON_DRIVER_EVENT_UNTETHERED_SUBSCRIPTION_STATE_CHANGE:
         {
             aeron_driver_agent_untethered_subscription_state_change_log_header_t *hdr =
-                (aeron_driver_agent_untethered_subscription_state_change_log_header_t *)message;
+                    (aeron_driver_agent_untethered_subscription_state_change_log_header_t *)message;
```

Comment:
Indent wrap is off. May need to fix your IDE.

Reply:
Fixed.